schemas/compat-data-schema.md

```diff
@@ -249,6 +249,26 @@ Examples:
 }
 ```
 
+### Initial versions
+
+The following table indicates initial versions for browsers in BCD. These are the earliest possible version numbers allowed to be used. These are not naturally "1" or "1.0" as explained in the notes column.
```

Comment:
I was slightly confused by the wording of the last sentence here. Does this still capture what you were getting at?

```suggestion
The following table indicates initial versions for browsers in BCD. These are the earliest possible version numbers allowed to be used. When the earliest version is not not naturally "1" or "1.0", see the _Notes_ column for an explanation.
```